bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/util/DefaultSchedulingBlock.scala

```diff
@@ -0,0 +1,50 @@
+package edu.gemini.spModel.util
+
+import java.util.Date
+
+import edu.gemini.pot.sp.ISPNode
+import edu.gemini.spModel.core.{Site, Semester, ProgramId}
+import edu.gemini.spModel.obs.SchedulingBlock
+
+import scalaz._, Scalaz._
+
+// N.B. this causes the object methods to appear as static methods on DefaultSchedulingBlock
+// which makes it easier to call from Java.
+sealed abstract class DefaultSchedulingBlock
+object DefaultSchedulingBlock {
+
+  /**
+   * Construct a default scheduling block for program in which `node` lives. This block will
+   * start at the center of the program semester OR the current semester, whichever is later,
+   * and will have no duration.
+   */
+  def forProgram(node: ISPNode): SchedulingBlock =
+    getSchedulingBlock(node, Site.GS)
```

Comment:
Coin flip. The point is just to get you near the middle of the semester, which is a point in time that's a few hours apart depending on the site. Doesn't matter here.